index.js

```diff
@@ -10,7 +10,10 @@ module.exports = {
           * Always delay requires, otherwise your plugin will cause trouble
           * with db-migrates performance and generates issues to your users.
           */
-        require('typescript').register();
+        var typescriptModule = require('typescript');
+        if (typescriptModule && typescriptModule.register) {
```

Comment:
@RemyJeancolas What exactly is the difference here? We do have guarantees that this module exists as it is part of the plugin. Or should this solve something else?